src/PSSparseServerTask.cpp

```diff
@@ -194,7 +194,7 @@ bool PSSparseServerTask::process_get_lr_sparse_model(
     uint32_t entry_index = load_value<uint32_t>(data);
     store_value<FEATURE_TYPE>(
         data_to_send_ptr,
-        lr_model->get_nth_weight(entry_index));
+        lr_model->get_nth_weight_nesterov(entry_index));
```

Comment:
As I understand this line is the only difference between SGD and Nesterov? If so we need to allow the use of both depending on the configuration.